_includes/key-points.html

```diff
@@ -24,10 +24,10 @@ <h2>The swiss-army knife of cryptocurrencies</h2>
     </div>
 
     <div class="medium-6 large-6 columns">
-      <h5 class="lead2">Disruptive E-commerce</h5>
-      <h2>World's most private marketplace</h2>
+      <h5 class="lead2">The Marketplace</h5>
+      <h2>The New Way of Doing eCommerce</h2>
```

Comment:
I would highlight the "private" part. "The new way" doesn't necessarity invoke anything better IMO.
Maybe something along the lines of:

> Peer to peer & private approach to e-commerce

?

Reply:
> Peer to peer & private approach to e-commerce

Sounds awesome for us, but does it really speak to the average user? We should make sure that our message can be something most people can relate to, hence why I'm trying to put more focus on how the marketplace will impact people. Usually, people relate more to saving money than being anonymous, in general. 

Let me think of something else, will post here. 

Thanks for the feedback on these.

Reply:
I don't see much problem with calling it what it is (even if technical) in these short titles. I see it as a technical term(s), which are then described "for average users" in the description below. This way it's also a bit educating, if that makes sense.
However if you think of anything better, I'm all for it!

Reply:
If so, please roll back your changes ;)